File: HackerRank/Python/Easy/E0060.py
# Problem Statement: https://www.hackerrank.com/challenges/standardize-mobile-number-using-decorators/problem

def wrapper(f):
    def fun(l):
    	l = map(lambda x: f'+91 {x[-10:-5]} {x[-5:]}', l)
    	f(l)
    return fun

File: HackerRank/Algorithms/Easy/E0060.cpp
/*
Problem Statement: https://www.hackerrank.com/challenges/string-construction/problem
*/

#include <iostream>
#include <string>
#include <set>

using namespace std;

int stringConstruction(string s) {
	return set<char>(s.begin(), s.end()).size();
}

int main()
{
	int n;
	string s;
	cin>>n;
	cin.ignore();
	for(int i=1 ; i<=n ; i++){
		getline(cin, s);
		cout<<stringConstruction(s)<<endl;
	}
	return 0;
}